server/src/main/java/org/opensearch/cluster/coordination/CoordinationMetadata.java

```diff
@@ -350,8 +350,8 @@ public String toString() {
     public static class VotingConfiguration implements Writeable, ToXContentFragment {
 
         public static final VotingConfiguration EMPTY_CONFIG = new VotingConfiguration(Collections.emptySet());
-        public static final VotingConfiguration MUST_JOIN_ELECTED_MASTER = new VotingConfiguration(
-            Collections.singleton("_must_join_elected_master_")
+        public static final VotingConfiguration MUST_JOIN_ELECTED_CLUSTER_MANAGER = new VotingConfiguration(
+            Collections.singleton("_must_join_elected_cluster_manager_")
```

Comment:
I assume this name is never used in a distributed manner where 1 node calls it `_must_join_elected_cluster_manager_` and the other `_must_join_elected_master_` and it's ok.

Reply:
Hi @dblock , thanks for your review! 
In my opinion, the preserved node id "_must_join_elected_master_" is only read in [ClusterFormationFailureHelper](https://github.com/opensearch-project/OpenSearch/blob/1.3.1/server/src/main/java/org/opensearch/cluster/coordination/ClusterFormationFailureHelper.java#L226), which result a line of log generated. The log says "master not discovered yet and this node was detached from its previous cluster", so it is never used in a distributed manner.
I will double check.

Reply:
Alright, to make it safe, it's better for me to deprecate `MUST_JOIN_ELECTED_MASTER` first in version 2.x, and accepting the both `_must_join_elected_cluster_manager_` and `_must_join_elected_master_` in `ClusterFormationFailureHelper`.
Otherwise, maybe doing a rolling upgrade from 2.x to 3.x after running `./bin/opensearch-node detach-cluster` will be a problem to identify the old identifier `_must_join_elected_master_`. Thank you! 😄